cdap-docs/integrations/source/apache-sentry.rst

```diff
@@ -8,6 +8,34 @@
 Apache Sentry
 =============
 
+.. _apache-sentry-configuration:
```

Comment:
This particular section (Apache Sentry configuration) is needed irrespective of whether you want to use the Sentry extension. It was moved from the Cloudera page. The rest of the doc, which talks about the authorization extension, is referenced from the README in the security extensions repo.